packages/libs/error-stack/Cargo.toml

```
@@ -1,7 +1,15 @@
 [package]
 name = "error-stack"
 version = "0.0.0"
+authors = ["HASH"]
 edition = "2021"
+rust-version = "1.61.0"
+description = "A context-aware error library with abritrary attached user data"
+documentation = "https://docs.rs/error-stack"
+readme = "README.md"
+repository = "https://github.com/hashintel/hash/tree/main/packages/libs/error-stack"
+keywords = ["error", "library", "stack", "no_std", "api"]
```

Perhaps `report` and `reporting`?

---

[crates.io only supports 5 keywords](https://doc.rust-lang.org/cargo/reference/manifest.html#the-keywords-field):

> Note: [crates.io](https://crates.io/) has a maximum of 5 keywords. Each keyword must be ASCII text, start with a letter, and only contain letters, numbers, `_` or `-`, and have at most 20 characters.

What do you think about this?
```suggestion
keywords = ["error", "library", "report", "no_std", "stack"]
```

---

Makes sense that they'd stop people spamming with keywords.  I like the suggestion 👍